src/lib/CLValue/Abstract.ts

```diff
@@ -34,14 +34,17 @@ export const resultHelper = <T, E>(
 };
 
 export abstract class CLType {
-  abstract toString(): string;
-  abstract toJSON(): any;
-  abstract linksTo: any;
+  abstract linksTo: string;
```

Comment:
I wanted to aske before, what's the purpose of linksTo?

Reply:
it was unused, but when designing we plan to use it for something else. so mainly a legacy thing, and now I used it to distinguish `tag` which is used to prepend byte that describes type from something that is mainly used internally to check the type